src/routes/routes.tsx

```
@@ -368,8 +366,43 @@
     errorElement: <ErrorPage />,
-    element: <About/>,
-  }
+    element: <About />,
+  },
+
+  // Admin routes
+
+  {
+    path: PATH_ADMIN.root,
+    errorElement: <ErrorPage />,
+    element: <PageWrapper children={<DashboardLayout />} />,
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)